docs/content/en/integrations/parsers/file/bearer_cli.md

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
I guess you have to rename bearer_cli.md to bearer.md to pass the unittests.

Reply:
Renamed everything to bearer_cli. This is the official name of the scanner.